virtual_ecosystem/models/plants/constants.py

```diff
@@ -11,6 +11,9 @@
 class PlantsConsts(ConstantsDataclass):
     """Constants for the :mod:`~virtual_ecosystem.models.plants` model."""
 
+    per_stem_annual_mortality_rate: float = 0.1
+    """Basic annual mortality rate for plants."""
+
```

Comment:
This is fussy - particularly in a temporary simple implementation - but a mortality rate is usually given as a number out of a given population size (5.7 / 1000 for example), where the `per_stem_annual_mortality_probability` is the probability per stem (so is in [0,1]). Probably better to stick with using `probability` not `rate`.